tests/test_cli_args.py

```diff
@@ -77,13 +78,16 @@ def test_defaults():
         ("extra-arg", '"-std=c++17 -Wall"', "extra_arg", ['"-std=c++17 -Wall"']),
         ("tidy-review", "true", "tidy_review", True),
         ("format-review", "true", "format_review", True),
+        ("jobs", "0", "jobs", None),
+        ("jobs", "1", "jobs", 1),
+        ("jobs", "4", "jobs", 4),
```

Comment:
To cover the exception raised in cli.py:
```suggestion
        ("jobs", "4", "jobs", 4),
        pytest.param("jobs", "x", "jobs", 0, marks=pytest.mark.xfail),
```